editor/editor_help.cpp

```diff
@@ -363,20 +362,19 @@ void EditorHelpIndex::_tree_item_selected() {
 		return;
 
 	emit_signal("open_class", s->get_text(0));
-
+	EditorNode::get_singleton()->call("_editor_select", EditorNode::EDITOR_SCRIPT);
 	hide();
 
 	//_goto_desc(s->get_text(0));
 }
 
 void EditorHelpIndex::select_class(const String &p_class) {
 
-	EditorNode *editor = EditorNode::get_singleton();
 	if (!tree_item_map.has(p_class))
 		return;
 	tree_item_map[p_class]->select(0);
 	class_list->ensure_cursor_is_visible();
-	editor->call("_editor_select", EditorNode::EDITOR_SCRIPT); // in case EditorHelpIndex beeen invoked on top of other editor window
+	EditorNode::get_singleton()->call("_editor_select", EditorNode::EDITOR_SCRIPT); // in case EditorHelpIndex beeen invoked on top of other editor window
```

Comment:
@nunodonato That bug exists before this PR. I think it is caused by this line. Commenting it out fixes the issue but I'm not sure what you mean in the code comment. If I understand correctly, `select_class` is called after search text is edited to select the most relevant result.